code/modules/urist/modules/wfc/modules/deepmaint/teleporters.dm

```diff
@@ -0,0 +1,143 @@
+
+/proc/deepmaint_yeet_trg(var/atom/movable/trg, var/xoffset = 1, var/yoffset = 1)
+	if(!istype(trg, /atom/movable))
+		return FALSE
+
+	if(isnull(trg.loc))
+		// No un-qdel-ing by accident!
+		return
+
+	if(istype(trg, /turf) || istype(trg, /area))
+		// This would be silly
+		return
+
+	if(istype(trg, /mob) && !istype(trg, DEEPMAINT_TELEPORTABLE_MOBTYPE))
```

Comment:
Not sure what DM does under the hood, but I'd hope using a built-in proc would be better for this?
```suggestion
	if(ismob(trg) && !istype(trg, DEEPMAINT_TELEPORTABLE_MOBTYPE))
```

Reply:
I didn't realize this was an actual builtin, since we have a couple of macros that pretend to be builtins.

Reply:
Yeah. All the _basic_ datums have built-in procs: `ismob()`, `isobj()`, `isturf()`, `isarea()` and also `ismovable()`. As you said, there _are_ a load that are just `istype` checks wearing coats, like `isAI()`